release-doc-in-process/all_vf.TTL

```diff
@@ -346,9 +366,87 @@ vf:fulfills     a owl:ObjectProperty ;
+        rdfs:range      xsd:boolean ;
+        rdfs:comment    "Defines if any resource of that type can be freely substituted for any other resource of that type when used, consumed, traded, etc." .
+
+#vf:unit a               owl:ObjectProperty ;
+#        rdfs:label      "unit" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:range      qudt:unit ;
+#        rdfs:comment    "The unit expected for this resource type as a default." .
+
+vf:definedQuantity a    owl:ObjectProperty ;
+        rdfs:label      "defined quantity" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      qudt:QuantityValue ;
+        rdfs:comment    "The amount and unit defined for this recipe flow." .
+
+# TODO: need to define this, something like currency, inventory, work, service; may be other aspects too.
+#vf:resourceCategory a    owl:ObjectProperty ;
+#        rdfs:label      "category" ;
+#        rdfs:domain     vf:ResourceType ;
+#        rdfs:comment    "Defines the category of behavior expected for the type of resource." .
+
+vf:developedBy a        owl:ObjectProperty ;
+        rdfs:label      "developed by" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      foaf:Agent ;
+        rdfs:comment    "The agent who developed this recipe." .
+
+vf:variantOf a           owl:ObjectProperty ;
+        rdfs:label      "variant of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is a variation of another recipe." .
+
+vf:successorOf a        owl:ObjectProperty ;
+        rdfs:label      "successor of" ;
+        rdfs:domain     vf:Recipe ;
+        rdfs:range      vf:Recipe ;
+        rdfs:comment    "This recipe is an updated version of another recipe." .
+
+vf:affected a           owl:ObjectProperty ;
+        rdfs:label      "affected" ;
+        rdfs:domain     vf:RecipeFlow ;
+        rdfs:range      vf:RecipeResource ;
+        rdfs:comment    "Defines the type of resource this recipe flow affects." .
+
+vf:hasDuration a        owl:ObjectProperty ;
+        rdfs:label      "duration" ;
+        rdfs:domain     vf:RecipeProcess ;
+        rdfs:range      time:TemporalDuration ;
+        rdfs:comment    "The calendar duration defined for this type of process." .
+
+vf:durationMultiplier 
+        a               owl:DataTypeProperty ;
+        rdfs:label      "multiplier" ;
+        rdfs:domain     vf:RecipeProcess ;
+        rdfs:range      ???? ; #TODO
+        rdfs:comment    "The calendar duration defined for this type of process." .
+
+vf:resourceBasedOn
+        a                   owl:ObjectProperty ;
+        rdfs:label          "resource typed as" ;
+        rdfs:domain         [ owl:unionOf (vf:Commitment vf:EconomicResource) ] ;
+        rdfs:range          vf:RecipeResource ;
+        rdfs:comment        "A resource's primary classification, part of describing that resource." .
 
-# something for models for GTIN, etc. identifiers
+vf:processBasedOn
```

Comment:
I think we can use same `vf:basedOn` between
* `vf:Process` and `vf:ProcessRecipe`
*  Flow (`vf:Intent` & `vf:Commitment`) and `vf:FlowRecipe`

Reply:
I guess that is the same as we did the other day with includedIn.  Are you sure?  The UML model will help because it will detail it properly.  But there will be no machine readable definition that is correct.  Possibly a larger topic, and it is in my mind because of the Protege issue.